test/WebSites/FormatterWebSite/Controllers/JsonFormatterController.cs

```diff
@@ -21,8 +22,11 @@ public IActionResult ReturnsIndentedJson()
                 Name = "John Williams"
             };
 
-            var jsonFormatter = new JsonOutputFormatter();
-            jsonFormatter.SerializerSettings.Formatting = Formatting.Indented;
+            var settings = new JsonSerializerSettings
+            {
+                Formatting = Formatting.Indented,
+            };
+            var jsonFormatter = new JsonOutputFormatter(settings, ArrayPool<char>.Shared);
```

Comment:
I doubt users often need to change serializer settings outside their `Startup` code. But, as I [said](https://github.com/aspnet/Mvc/pull/4492/files#r59992980) in `JsonOutputFormatter`, I could restore the `JsonOutputFormatter(JsonSerializerSettings)` constructor overload.